src/packages/@app/pages/CollectionExplorerPage/CollectionExplorerPage.ViewModel.ts

```diff
@@ -63,7 +62,7 @@ class CollectionExplorerPage {
   public getUserRoleInWorspace = async () => {
     let role: WorkspaceRole;
     const userWorkspaceLevelRoleSubscribe = userWorkspaceLevelRole.subscribe(
-      (value: any) => {
+      (value) => {
         role = WorkspaceRole.WORKSPACE_ADMIN;
```

Comment:
here we are returning admin always, can we change this to the actual role or if we are not using this function anywhere, we can also remove it.

Reply:
resolved